modules/pubxaiAnalyticsAdapter.md

```diff
@@ -0,0 +1,20 @@
+# Overview
+Module Name: PubX.io Analytics Adapter
+Module Type: Analytics Adapter
+Maintainer: phaneendra@pubx.ai
+
+# Description
+
+Analytics adapter for prebid provided by Pubx.ai. Contact alex@pubx.ai for information.
+
+# Test Parameters
+
+```
+{
+  provider: 'pubxai',
+    options : {
```

Comment:
I don't see `samplingRate` mentioned here. Also, I think it would be useful to describe and document all the params here (in a separate section). You can add some table with property name, type, is it required, description and example, what do you think?